src/components/emojis.ts

```diff
@@ -0,0 +1,12 @@
+import { Emoji } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+
+let client: CommandoClient;
+
+export const initEmotes = (cli: CommandoClient): void => {
+  client = cli;
+};
+
+export const getEmoteByName = (name: string): Emoji | undefined => {
```

Comment:
Oh and streamline the naming conventions of the functions in this file also! (-> `initEmojis`)